repo2docker/buildpacks/base.py

```diff
@@ -117,6 +117,7 @@
 # Copy and chown stuff. This doubles the size of the repo, because
 # you can't actually copy as USER, only as root! Thanks, Docker!
 USER root
+RUN mkdir "${HOME}" && chown ${NB_USER}:${NB_USER} "${HOME}"
```

Comment:
```suggestion
# Explicitly create $HOME, in case $REPO_DIR is explicitly set to be somewhere
RUN mkdir "${HOME}" && chown ${NB_USER}:${NB_USER} "${HOME}"
```